tests/drivers/spi/spim_instances/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,262 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+&pinctrl {
+	spi120_default_test: spi120_default_test {
+		group1 {
+			psels = <NRF_PSEL(SPIM_MISO, 7, 6)>;
+		};
+		group2 {
```

Comment:
if these are expected to be upstreamed in future then newlines needed between child nodes